actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -654,6 +414,18 @@ private Task onInvocation(final HandlerContext ctx, final Invocation invocation)
             return Task.done();
         }
 
+        // Invocation is @OnlyIfActivated but got sent despite actor deactivation (this is possible even in a healthy
```

Comment:
This is the secondary bugfix mentioned elsewhere. There was a case where an `Invocation` for an `@OnlyIfActivated` method could be dispatched from a source stage A while the target actor was believed to be active on destination stage B. However, that actor could have deactivated before the `Invocation` arrived, and would prompt the destination stage B to start that actor -- even though the `Invocation` specifically says `@OnlyIfActivated`. So we need to guard against both _creating_ and _receiving_ those `Invocation`s.

See the new test case in `OnlyIfActivatedTest`.